### PR TITLE
Update reference links

### PR DESCRIPTION
**Pull request by javagl**
To be merged after https://github.com/KhronosGroup/glTF-Tutorials/pull/66

Fixes https://github.com/KhronosGroup/glTF-Tutorials/issues/59

### CODE REVIEW

**Comment by javagl** [comment]
@emackey This is the third of three pull requests

- https://github.com/KhronosGroup/glTF-Tutorials/pull/65 
- https://github.com/KhronosGroup/glTF-Tutorials/pull/66
- This one

They build on top of each other: The first ones are rather simple and small, and this one is rather technical (in that it _mainly_ consisted of a "replace all"), so I thought that it wouldn't make much sense to have them as separate PRs, always starting from `master`.

Depending on the preferred process, it could make sense to review and merge *this* one, implicitly dropping the two other ones.

File: gltfTutorial/gltfTutorial_005_BuffersBufferViewsAccessors.md [hunk 1]
```diff
@@ -29,7 +29,7 @@ Parts of the data of a `buffer` may have to be passed to the renderer as vertex
 
 ## BufferViews
 
-The first step of structuring the data from a `buffer` is with [`bufferView`](https://github.com/KhronosGroup/glTF/tree/master/specification/2.0/#reference-bufferView) objects. A `bufferView` represents a "slice" of the data of one buffer. This slice is defined using an offset and a length, in bytes. The [minimal glTF file](gltfTutorial_003_MinimalGltfFile.md) defined two `bufferView` objects:
+The first step of structuring the data from a `buffer` is with [`bufferView`](https://www.khronos.org/registry/glTF/specs/2.0/glTF-2.0.html#reference-bufferView) objects. A `bufferView` represents a "slice" of the data of one buffer. This slice is defined using an offset and a length, in bytes. The [minimal glTF file](gltfTutorial_003_MinimalGltfFile.md) defined two `bufferView` objects:
```
Comment (emackey) [inline gltfTutorial/gltfTutorial_005_BuffersBufferViewsAccessors.md:32]:
```suggestion
The first step of structuring the data from a `buffer` is with [`bufferView`](https://www.khronos.org/registry/glTF/specs/2.0/glTF-2.0.html#reference-bufferview) objects. A `bufferView` represents a "slice" of the data of one buffer. This slice is defined using an offset and a length, in bytes. The [minimal glTF file](gltfTutorial_003_MinimalGltfFile.md) defined two `bufferView` objects:
```

**Review by emackey — COMMENTED** [review]
A few URL tweaks, but this is great @javagl.  Thanks!

**Comment by javagl** [comment]
Thanks @emackey ! I had done a "full text search", and a _largely_ automated replace, with manual tweaks for some critical ones (e.g. some of these `#nested-anchors-with-dashes`). I have no idea how I missed the ones that still obviously pointed to GitHub, and things like the `bufferView` vs. `bufferview` are ... subtle, but I think that most if not all links should be correct now. (If someone spots a broken one, that could be a dedicated issue/PR).

**Pull request merged**